package.json

```diff
@@ -12,6 +12,8 @@
     "@mui/material": "^5.10.7",
     "@mui/styles": "^5.10.7",
     "@mui/x-date-pickers": "^5.0.3",
+    "add-to-calendar-button": "^2.2.9",
+    "add-to-calendar-button-react": "^2.2.7",
```

Comment:
I looked at these repos.  the -react version is a small wrapper around the base.  Both appear to be actively developed and supported. The -react repo is a one-person (jekuer) effort since last Oct, but is ~100 lines and wouldn't be hard to pull inside.

add-to-calendar-button has been actively developed for over 2 years, with multiple developers, but with the -react developer as the most prolific.  It appears to contain about 2000-3000 lines of code.  It has >1100 stars and >19,000 weekly npm installs, so it's fairly widely used.  The lead developer (jekuer) say he is in Germany.   There's a simple website, add-to-calendar-button.com.

This all looks reasonable to me, with no red flags, though it's still a small team effort, not one from a well-known organization.

We looked at solving this problem (creating links to standard calendars) a few years ago, and it was too much work for the time available.  So it's nice to find a package to do it.

Reply:
Awesome, thanks for looking into it!